db/migrate/20220119150831_add_column_to_greetings.rb

```diff
@@ -0,0 +1,5 @@
+class AddColumnToGreetings < ActiveRecord::Migration[7.0]
+  def change
+    add_column :greetings, :greeting, :string
+  end
+end
```

Comment:
- [ ] I suggest that you delete this migration file if you are not using it because I am not able to run migration

Reply:
Thank you for your feedback! I deleted the migration. ✌️  ✌️